### ISSUE TITLE
Why did you create this entity? what do you mean by status? is it like and dislike? 

### ISSUE THREAD

**Issue by vahid-forghani** [amezenin/encyclopedia-kn#2]
https://github.com/amezenin/encyclopedia-kn/blob/c71915518d43bdb8942f420fb19acd7cd83ed6c5/src/main/java/com/knits/product/model/VoteArticle.java#L14

**Comment by amezenin** [comment]
> https://github.com/amezenin/encyclopedia-kn/blob/c71915518d43bdb8942f420fb19acd7cd83ed6c5/src/main/java/com/knits/product/model/VoteArticle.java#L14

yes, I thought that a user can like and dislike a comment or a post separately. and perhaps additional information will be needed in the middle table - creation date, etc.

**Comment by amezenin** [comment]
fixed
